code/modules/mob/living/simple_animal/hostile/alien.dm

```diff
@@ -23,8 +23,8 @@
 	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
 	unsuitable_atmos_damage = 15
 	heat_damage_per_tick = 20
-	pressure_resistance = 50    //50 kPa difference required to push
-	throw_pressure_limit = 100  //100 kPa difference required to throw
+	pressure_resistance = 100    //100 kPa difference required to push
+	throw_pressure_limit = 120   //120 kPa difference required to throw

```

Comment:
It was pretty silly being stuck in space as a xeno queen. Felt like a right lemon. 👍